homeassistant/components/denonavr/__init__.py

```diff
@@ -60,7 +61,6 @@ async def async_setup_entry(
 
     # Connect to receiver
```

Comment:
We have a httpx helper
`from homeassistant.helpers.httpx_client import get_async_client`

Reply:
This could be used instead of `httpx.AsyncClient` in denonavr package itself right?

Reply:
yes